tasks/init.json

```diff
@@ -0,0 +1,52 @@
+{
+  "description": "Make a HTTP or HTTPS request.",
+  "parameters": {
+    "body": {
+      "description": "The request body.",
+      "type": "Optional[Data]"
+    },
+    "body_format": {
+      "description": "The format of the request body. Automatically attempts to convert the body to the specified format.",
+      "type": "Enum[form-urlencoded, json, raw]",
+      "default": "json"
+    },
+    "cacert": {
+      "description": "An absolute path to the CA certificate.",
+      "type": "Optional[String[1]]"
+    },
+    "cert": {
+      "description": "An absolute path to the client certificate.",
+      "type": "Optional[String[1]]"
+    },
+    "headers": {
+      "description": "A map of headers to add to the payload.",
+      "type": "Optional[Hash]"
+    },
+    "key": {
+      "description": "An absolute path to the RSA keypair.",
+      "type": "Optional[String[1]]"
+    },
+    "method": {
+      "description": "The HTTP method to use.",
+      "type": "Enum[delete, get, post, put]",
+      "default": "get"
+    },
+    "success_codes": {
+      "description": "A list of status codes that indicate the task should return a successful result.",
+      "type": "Array[Numeric]",
+      "default": [200]
+    },
+    "url": {
+      "description": "The fully qualified URL scheme to make requests to.",
+      "type": "String[1]"
+    },
+    "user_agent": {
```

Comment:
Added this since it seems helpful for users who may want to log when Bolt is making requests.